Mathlib/Dynamics/Flow.lean

```diff
@@ -164,6 +166,8 @@ def toHomeomorph (t : τ) : (α ≃ₜ α) where
   invFun := ϕ (-t)
   left_inv x := by rw [← map_add, neg_add_cancel, map_zero_apply]
   right_inv x := by rw [← map_add, add_neg_cancel, map_zero_apply]
+  continuous_toFun := by fun_prop
+  continuous_invFun := by fun_prop
```

Comment:
Is this for performance? If so, resolve this convo.

Reply:
Yes, this is just for performance. This is a purely hypothetical changes; this definition is not used in mathlib yet - I'm happy to revert it also.